### PR TITLE
Spawning transpiler

### PR DESCRIPTION
**Pull request by VALERA771**
No errors on startup, will test when new version will be released on clients

### CODE REVIEW

**Comment by louis1706** [comment]
> No errors on startup, will test when new version will be released on clients

Yeah i will not merge this before test lol

**Comment by VALERA771** [comment]
Tested. Working fine

File: Exiled.Events/Patches/Events/Player/Spawning.cs [hunk 1]
```diff
@@ -37,38 +34,85 @@ private static MethodInfo TargetMethod()
...
+                    // oldRole
+                    new(OpCodes.Ldarg_2),
 
-                    Handlers.Player.OnSpawning(ev);
+                    // SpawningEventArgs(Player, Vector3, 0, PlayerRoleBase)
+                    new(OpCodes.Newobj, GetDeclaredConstructors(typeof(SpawningEventArgs))[0]),
 
-                    hub.transform.position = ev.Position;
-                    fpcRole.FpcModule.MouseLook.CurrentHorizontal = ev.HorizontalRotation;
-                    hub.connectionToClient.Send(new FpcOverrideMessage(ev.Position, ev.HorizontalRotation), 0);
```
Comment (louis1706) [inline Exiled.Events/Patches/Events/Player/Spawning.cs:61]: Where is this call ?

Comment (louis1706) [reply Exiled.Events/Patches/Events/Player/Spawning.cs:61]: if you can verify than ev.HorizontalRotation work without this ?

Comment (VALERA771) [reply Exiled.Events/Patches/Events/Player/Spawning.cs:61]: I don't really think that it's neccecary cuz game is also changing it (bruh yes, I completly forgot that game will override it again)

Comment (louis1706) [reply Exiled.Events/Patches/Events/Player/Spawning.cs:61]: yeah but wasn't working with base game that why i ask

**Comment by VALERA771** [comment]
I've decided to fully rewrite transpiler so it'll replace original method (like prefix do)

**Review by louis1706 — APPROVED** [review]

**Review by louis1706 — APPROVED** [review]

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Pull request merged**